src/index.tsx

```diff
@@ -255,6 +256,7 @@ function UseWalletProvider({
     throw new Error('<UseWalletProvider /> has already been declared.')
   }
 
+  const [chainId, setChainId] = useState<number>(1)
```

Comment:
Should be `-1` right?
```suggestion
  const [chainId, setChainId] = useState<number>(-1)
```